plugins/modules/zos_mvs_raw.py

```diff
@@ -3090,6 +3095,43 @@ def get_content(formatted_name, binary=False, from_encoding=None, to_encoding=No
         return stdout
 
 
+def modify_contents(contents):
+    """Return the content of dd_input to a valid form for a JCL program.
+
+    Args:
+        contents (str or list): The string or list with the program.
+
+    Returns:
+        contents: The content in a proper multi line str.
+    """
+    if not isinstance(contents, list):
+        if len(contents) > 1 and contents[0] != " " and contents[1] != " ":
+            contents = "  {0}".format(contents)
```

Comment:
Would there ever be a case where only 1 space is there?  This caught my eye. but I don't think it needs changed.

Reply:
Could happen, but by the recommendations is to put 2 spaces